Model/Order/Creditmemo/ItemPlugin.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * @by SwiftOtter, Inc. 4/19/17
+ * @website https://swiftotter.com
+ **/
+
+namespace SwiftOtter\ShippingSurcharge\Model\Order\Creditmemo;
+
+use SwiftOtter\ShippingSurcharge\Model\Surcharge as SurchargeModel;
+
+class ItemPlugin
+{
+    public function afterCalcRowTotal(\Magento\Sales\Model\Order\Creditmemo\Item $item)
```

Comment:
So, that's a bit tricky. Since this is a plugin, I do like putting the FQDN there as well; that's a great point. I would say aliasing is always worth it if it enhances the readability of the code. I really like aliasing for clarity, like this: `use Magento\Sales\Model\Order\Creditmemo\Item as CreditmemoItem`. I feel like this is the best of both worlds: it avoids ambiguity, while increasing readability.